### PR TITLE
Update tuning rules to exclude test set from scoring procedure

### PR DESCRIPTION
**Pull request by fsschneider**
As discussed in the WG meetings, this PR updates the rules to use "Option C", i.e. removing the test set from the scoring procedure and instead only using the validation set.

The test targets are still defined for each workload and will be used for additional analysis of the results but they are not relevant for scoring/ranking submissions.

Merge after #476

### CODE REVIEW

**Review by priyakasimbeg — DISMISSED** [review]

**Review by georgedahl — DISMISSED** [review]
Some of these things need to change, but maybe we should just merge this PR and do another after it? For example, just merge this and then make it clear the submission process information is just documentation?

**Review by georgedahl — APPROVED** [review]

**Review by priyakasimbeg — APPROVED** [review]

**Pull request merged**
